integration-cli/docker_cli_exec_test.go

```diff
@@ -193,10 +200,16 @@ func (s *DockerCLIExecSuite) TestExecTTYWithoutStdin(c *testing.T) {
 		}
 
 		expected := "the input device is not a TTY"
-		if runtime.GOOS == "windows" {
-			expected += ".  If you are using mintty, try prefixing the command with 'winpty'"
-		}
```

Comment:
This didn't add value; the extra hint is added in the CLI, so we only need to check for the first part of the error; https://github.com/docker/cli/blob/5d8fb335d4e0db3a7250892de0e07b1fa7d3a730/cli/streams/in.go#L46-L52